src/Workspaces/Core/Portable/FindSymbols/SyntaxTree/SyntaxTreeIndex_Forwarders.cs

```diff
@@ -19,20 +19,21 @@ internal sealed partial class SyntaxTreeIndex
         public bool ProbablyContainsStringValue(string value) => _literalInfo.ProbablyContainsStringValue(value);
         public bool ProbablyContainsInt64Value(long value) => _literalInfo.ProbablyContainsInt64Value(value);
 
-        public bool ContainsForEachStatement => _contextInfo.ContainsForEachStatement;
-        public bool ContainsDeconstruction => _contextInfo.ContainsDeconstruction;
         public bool ContainsAwait => _contextInfo.ContainsAwait;
+        public bool ContainsBaseConstructorInitializer => _contextInfo.ContainsBaseConstructorInitializer;
+        public bool ContainsConversion => _contextInfo.ContainsConversion;
+        public bool ContainsDeconstruction => _contextInfo.ContainsDeconstruction;
+        public bool ContainsElementAccessExpression => _contextInfo.ContainsElementAccessExpression;
+        public bool ContainsForEachStatement => _contextInfo.ContainsForEachStatement;
+        public bool ContainsGlobalKeyword => _contextInfo.ContainsGlobalKeyword;
+        public bool ContainsGlobalSuppressMessageAttribute => _contextInfo.ContainsGlobalSuppressMessageAttribute;
         public bool ContainsImplicitObjectCreation => _contextInfo.ContainsImplicitObjectCreation;
+        public bool ContainsIndexerMemberCref => _contextInfo.ContainsIndexerMemberCref;
         public bool ContainsLockStatement => _contextInfo.ContainsLockStatement;
-        public bool ContainsUsingStatement => _contextInfo.ContainsUsingStatement;
         public bool ContainsQueryExpression => _contextInfo.ContainsQueryExpression;
         public bool ContainsThisConstructorInitializer => _contextInfo.ContainsThisConstructorInitializer;
-        public bool ContainsBaseConstructorInitializer => _contextInfo.ContainsBaseConstructorInitializer;
-        public bool ContainsElementAccessExpression => _contextInfo.ContainsElementAccessExpression;
-        public bool ContainsIndexerMemberCref => _contextInfo.ContainsIndexerMemberCref;
         public bool ContainsTupleExpressionOrTupleType => _contextInfo.ContainsTupleExpressionOrTupleType;
-        public bool ContainsGlobalSuppressMessageAttribute => _contextInfo.ContainsGlobalSuppressMessageAttribute;
-        public bool ContainsConversion => _contextInfo.ContainsConversion;
+        public bool ContainsUsingStatement => _contextInfo.ContainsUsingStatement;
```

Comment:
just sorted for my sanity.  and `public bool ContainsGlobalKeyword => _contextInfo.ContainsGlobalKeyword;` was added.